server/app/routes/api/users.js

```diff
@@ -1,15 +1,23 @@
 var express = require('express');
 var router = express.Router();
 var mongoose = require('mongoose');
+require('../../../db/models/user');
+var User = mongoose.model('User');
 
-router.use('/:id/reviews', require('./reviews'));
-//TW Add router.param() --see reviews for example
-//TW Where are your nexts coming from?
+router.use('/:userId/reviews', require('./reviews'));
+
+router.param('userId', function (req, res, next, userId){
+  User.findById(userId)
+  .then(function(user){
+    req.reqUser = user;
```

Comment:
We are only doing a sanitize on the get route, so I'm going to just leave it there.